<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterWindow</class>
 <widget class="QDialog" name="RegisterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>720</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>550</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
background-image: url(:/Icons/ProgramIcons/bg.png);
}

QLineEdit {
	font: 9pt &quot;Joan&quot;;
    border: 2px solid gray;
    border-radius: 10px;
	border-color: rgb(13, 54, 101);
     padding:4px 20px 4px 4px;
    background: white;
    selection-background-color: darkgray;
}

QPushButton{
font: bold 10pt  &quot;Heebo&quot;;
background-color: rgb(13, 54, 101);
color: rgb(233, 236, 237);
border-radius: 10px;
padding: 5px;
}
QPushButton::hover{
border-style: inset;
background-color:rgb(25, 104, 195);
color: rgb(213, 173, 18);

}
QPushButton::pressed{
border: 2px solid rgb(0, 170, 0);
}

QComboBox{
	font: 9pt &quot;Joan&quot;;
    border: 2px solid gray;
    border-radius: 10px;
	border-color: rgb(13, 54, 101);
     padding:4px 20px 4px 4px;
    background: white;
    selection-background-color: darkgray;
}
QComboBox::drop-down 
{
    border: 0px;
}
QComboBox::down-arrow{
   
	image: url(:/Icons/ProgramIcons/downarrow.png);
    width: 10px;
    height: 10px;
    }

QListView{
font: 9pt &quot;Joan&quot;;
    border: 2px solid gray;
    border-radius: 10px;
	border-color: rgb(13, 54, 101);
     padding:4px 20px 4px 4px;
    background: white;
    selection-background-color: darkgray;
}

QRadioButton::indicator::unchecked{ 
border: 1px solid; 
border-color: rgb(13, 54, 101);
border-radius: 9px;
border-width: 2px;
background-color: white; 
width: 15px; 
height: 15px; 
}

QRadioButton::indicator::checked{ 
border-color: rgb(13, 54, 101);
background-color:  rgb(213, 173, 18);
border: 1px solid; 
border-radius: 9px;
border-width: 2px;
width: 15px; 
height: 15px; 
}

QScrollBar:vertical {
border: none;
	background-color: rgb(208, 244, 255);
width: 14px;
margin: 15px 0 15px 0;
border-radius: 7px;
}


/* HANDLE BAR VERTICAL */
QScrollBar::handle:vertical {
	background-color: rgb(12, 12, 84);
min-height: 30px;
border-radius: 7px;
}
QScrollBar::handle:vertical:hover{
background-color:rgb(28, 114, 206)
}
QScrollBar::handle:vertical:pressed {
	background-color: rgb(8, 32, 59);
}


/* BTN TOP - SCROLLBAR */
QScrollBar::sub-line:vertical {
border: none;
background-color: rgb(213, 173, 18);
height: 15px;
border-top-left-radius: 7px;
border-top-right-radius: 7px;
subcontrol-position: top;
subcontrol-origin: margin;
}
QScrollBar::sub-line:vertical:hover {
background-color: rgb(211, 213, 66);
}
QScrollBar::sub-line:vertical:pressed {
	background-color: rgb(139, 111, 12);
}


/* BTN BOTTOM - SCROLLBAR */
QScrollBar::add-line:vertical {
border: none;
background-color: rgb(213, 173, 18);
height: 15px;
border-bottom-left-radius: 7px;
border-bottom-right-radius: 7px;
subcontrol-position: bottom;
subcontrol-origin: margin;
}
QScrollBar::add-line:vertical:hover {
background-color: rgb(211, 213, 66);
}
QScrollBar::add-line:vertical:pressed {
background-color: rgb(139, 111, 12);
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item row="11" column="2">
      <widget class="QLabel" name="LastNameLabel">
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;Joan&quot;;</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Last Name&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="10" column="2">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="12" column="0">
      <widget class="QLineEdit" name="FirstNameLineEdit"/>
     </item>
     <item row="10" column="1">
      <widget class="QLineEdit" name="PasswordLineEdit">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="FirstNameLabel">
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;Joan&quot;;</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;First Name&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QLineEdit" name="MiddleNameLineEdit"/>
     </item>
     <item row="1" column="0" colspan="3">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="16" column="2">
      <widget class="QWidget" name="widget_3" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QRadioButton" name="GenderFemaleButton">
          <property name="styleSheet">
           <string notr="true">font: 9pt &quot;Joan&quot;;</string>
          </property>
          <property name="text">
           <string>Female</string>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="GenderMaleButton">
          <property name="styleSheet">
           <string notr="true">font: 9pt &quot;Joan&quot;;</string>
          </property>
          <property name="text">
           <string>Male</string>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="14" column="1">
      <widget class="QLineEdit" name="AcademicEmailLineEdit"/>
     </item>
     <item row="2" column="0" colspan="3" alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="RegisterLabel">
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;Joan&quot;;</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600; color:#0d3665;&quot;&gt;REGISTER&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QLabel" name="MiddleNameLabel">
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;Joan&quot;;</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Middle Name&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="UsernameLabel">
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;Joan&quot;;</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Username&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="18" column="2">
      <widget class="QPushButton" name="SignInButton">
       <property name="text">
        <string>Sign-In Instead</string>
       </property>
      </widget>
     </item>
     <item row="14" column="0">
      <widget class="QLineEdit" name="PersonalEmailLineEdit"/>
     </item>
     <item row="12" column="2">
      <widget class="QLineEdit" name="LastNameLineEdit"/>
     </item>
     <item row="14" column="2">
      <widget class="QLineEdit" name="ContactNumberLineEdit"/>
     </item>
     <item row="15" column="0">
      <widget class="QLabel" name="CourseLabel">
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;Joan&quot;;</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Course/Department/Organization&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <widget class="QLabel" name="AcademicEmailLabel">
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;Joan&quot;;</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;PLM Email&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="18" column="1" alignment="Qt::AlignRight">
      <widget class="QLabel" name="SignInLabel">
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;Joan&quot;;</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Already have an account?&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="19" column="1">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="10" column="0">
      <widget class="QLineEdit" name="UsernameLineEdit"/>
     </item>
     <item row="17" column="0" colspan="3">
      <widget class="QWidget" name="widget" native="true">
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item row="2" column="0">
         <widget class="QLineEdit" name="AvailableTagsSearch"/>
        </item>
        <item row="4" column="0" colspan="2">
         <widget class="QListView" name="AvailableTagsList">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QLabel" name="AvailableTagsLabel">
          <property name="styleSheet">
           <string notr="true">font: 9pt &quot;Joan&quot;;</string>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Available Tags&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2" rowspan="2">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="5" rowspan="3">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="3" rowspan="3" colspan="2">
         <widget class="QListView" name="YourTagsList">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QLabel" name="YourTagsLabel">
          <property name="styleSheet">
           <string notr="true">font: 9pt &quot;Joan&quot;;</string>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Your Tags&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="AddTagButton">
          <property name="text">
           <string>Add Tag</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QPushButton" name="RemoveYourTag">
          <property name="text">
           <string>Remove</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="16" column="0" colspan="2">
      <widget class="QComboBox" name="CourseLineEdit">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QLabel" name="PersonalEmailLabel">
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;Joan&quot;;
font-weight:600;</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Personal Email&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLabel" name="PasswordLabel">
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;Joan&quot;;</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Password&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="13" column="2">
      <widget class="QLabel" name="ContactNumberLabel">
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;Joan&quot;;
font-weight:600;</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Contact Number&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0" colspan="2">
      <widget class="QWidget" name="widget_2" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item alignment="Qt::AlignHCenter">
         <widget class="QRadioButton" name="AccountTypeGuestButton">
          <property name="styleSheet">
           <string notr="true">font: 9pt &quot;Joan&quot;;</string>
          </property>
          <property name="text">
           <string>Non-Student (of PLM)</string>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QRadioButton" name="AccountTypeStudentButton">
          <property name="styleSheet">
           <string notr="true">font: 9pt &quot;Joan&quot;;</string>
          </property>
          <property name="text">
           <string>Student</string>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QRadioButton" name="AccountTypeFacultyButton">
          <property name="styleSheet">
           <string notr="true">font: 9pt &quot;Joan&quot;;</string>
          </property>
          <property name="text">
           <string>Staff</string>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="AccountTypeLabel">
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;Joan&quot;;</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;I am a:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="15" column="2">
      <widget class="QLabel" name="GenderLabel">
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;Joan&quot;;</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Gender&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="18" column="0">
      <widget class="QPushButton" name="CreateAccountButton">
       <property name="text">
        <string>Create Account</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="3">
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0,0">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="ProfilePictureLabel">
         <property name="text">
          <string>asdasd</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ProfilePictureButton">
         <property name="text">
          <string>Select Profile Picture</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>AccountTypeGuestButton</tabstop>
  <tabstop>AccountTypeStudentButton</tabstop>
  <tabstop>AccountTypeFacultyButton</tabstop>
  <tabstop>UsernameLineEdit</tabstop>
  <tabstop>PasswordLineEdit</tabstop>
  <tabstop>FirstNameLineEdit</tabstop>
  <tabstop>MiddleNameLineEdit</tabstop>
  <tabstop>LastNameLineEdit</tabstop>
  <tabstop>PersonalEmailLineEdit</tabstop>
  <tabstop>AcademicEmailLineEdit</tabstop>
  <tabstop>ContactNumberLineEdit</tabstop>
  <tabstop>CourseLineEdit</tabstop>
  <tabstop>GenderFemaleButton</tabstop>
  <tabstop>GenderMaleButton</tabstop>
  <tabstop>AvailableTagsSearch</tabstop>
  <tabstop>AddTagButton</tabstop>
  <tabstop>YourTagsList</tabstop>
  <tabstop>RemoveYourTag</tabstop>
  <tabstop>AvailableTagsList</tabstop>
  <tabstop>CreateAccountButton</tabstop>
  <tabstop>SignInButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
